internal/app/triggermessageprocessor.go

```diff
@@ -155,3 +197,11 @@ func (mp *triggerMessageProcessor) MessageReceived(ctx interfaces.AppFunctionCon
 
 	return finalErr
 }
+
+// ReceivedInvalidMessage is called when an error occurs decoding the message from the MessageBus into the MessageEnvelope
+func (mp *triggerMessageProcessor) ReceivedInvalidMessage() {
+	// messagesReceived includes all message received
+	mp.messagesReceived.Inc(1)
```

Comment:
I guess this answers my question above. I wonder if that should maybe added as a comment somewhere?

Reply:
Planning to add clarification in the documentation when it is updated with all the new metrics. Will also add comment in the struct where it is defined.

Reply:
Comment added